### PR TITLE
Updated details of Arduino MKR1000

### PR DESCRIPTION
**Pull request by animeshsrivastava24**
Added the details of PWM Pin and Frequency of operation. In response to Issue #587

### CODE REVIEW

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 1]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
 
 [float]
 === Description
-Writes an analog value (http://arduino.cc/en/Tutorial/PWM[PWM wave]) to a pin. Can be used to light a LED at varying brightnesses or drive a motor at various speeds. After a call to `analogWrite()`, the pin will generate a steady square wave of the specified duty cycle until the next call to `analogWrite()` (or a call to `digitalRead()` or `digitalWrite()`) on the same pin. The frequency of the PWM signal on most pins is approximately 490 Hz. On the Uno and similar boards, pins 5 and 6 have a frequency of approximately 980 Hz.
+Writes an analog value (http://arduino.cc/en/Tutorial/PWM[PWM wave]) to a pin. Can be used to light a LED at varying brightnesses or drive a motor at various speeds. After a call to `analogWrite()`, the pin will generate a steady square wave of the specified duty cycle until the next call to `analogWrite()` (or a call to `digitalRead()` or `digitalWrite()`) on the same pin. 
+[options="header"]
+|================================================================================================================================================
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
|====================================================================================================
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 2]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+| Board                | PWM Pins                         | PWM Frequency
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| Board                | PWM Pins                        | PWM Frequency
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 3]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+Writes an analog value (http://arduino.cc/en/Tutorial/PWM[PWM wave]) to a pin. Can be used to light a LED at varying brightnesses or drive a motor at various speeds. After a call to `analogWrite()`, the pin will generate a steady square wave of the specified duty cycle until the next call to `analogWrite()` (or a call to `digitalRead()` or `digitalWrite()`) on the same pin. 
+[options="header"]
+|================================================================================================================================================
+| Board                | PWM Pins                         | PWM Frequency                    
+| Uno, Nano, Mini      | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 5 and 6(980 Hz)
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| Uno, Nano, Mini      | 3, 5, 6, 9, 10, 11              | 490 Hz (pins 5 and 6: 980 Hz)
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 4]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+| Mega                 | 2 - 13 and 44 - 46               | 490 Hz, exception pin 4 and 13(980 Hz)
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| Mega                 | 2 - 13, 44 - 46                 | 490 Hz (pins 4 and 13: 980 Hz)
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 5]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+| Leonardo, Micro, Yún | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 3 and 11(980 Hz)
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| Leonardo, Micro, Yún | 3, 5, 6, 9, 10, 11, 13          | 490 Hz (pins 3 and 11: 980 Hz)
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 6]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+Writes an analog value (http://arduino.cc/en/Tutorial/PWM[PWM wave]) to a pin. Can be used to light a LED at varying brightnesses or drive a motor at various speeds. After a call to `analogWrite()`, the pin will generate a steady square wave of the specified duty cycle until the next call to `analogWrite()` (or a call to `digitalRead()` or `digitalWrite()`) on the same pin. 
+[options="header"]
+|================================================================================================================================================
+| Board                | PWM Pins                         | PWM Frequency                    
+| Uno, Nano, Mini      | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 5 and 6(980 Hz)
+| Mega                 | 2 - 13 and 44 - 46               | 490 Hz, exception pin 4 and 13(980 Hz)
+| Leonardo, Micro, Yún | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 3 and 11(980 Hz)                
+| Uno WiFi Rev.2       | 3, 5, 6, 9 and 10                | 490 Hz
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| Uno WiFi Rev.2       | 3, 5, 6, 9, 10                  | 976 Hz
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 7]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+Writes an analog value (http://arduino.cc/en/Tutorial/PWM[PWM wave]) to a pin. Can be used to light a LED at varying brightnesses or drive a motor at various speeds. After a call to `analogWrite()`, the pin will generate a steady square wave of the specified duty cycle until the next call to `analogWrite()` (or a call to `digitalRead()` or `digitalWrite()`) on the same pin. 
+[options="header"]
+|================================================================================================================================================
+| Board                | PWM Pins                         | PWM Frequency                    
+| Uno, Nano, Mini      | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 5 and 6(980 Hz)
+| Mega                 | 2 - 13 and 44 - 46               | 490 Hz, exception pin 4 and 13(980 Hz)
+| Leonardo, Micro, Yún | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 3 and 11(980 Hz)                
+| Uno WiFi Rev.2       | 3, 5, 6, 9 and 10                | 490 Hz            
+| MKR boards           | 0, 1, 2, 3, 4, 5, 6, 7, 8, 10, A3 - or 18 -, A4 -or 19                           | 732 Hz
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| MKR boards *         | 0 - 8, 10, A3 (18), A4 (19)     | 732 Hz
| MKR1000 WiFi *       | 0 - 8, 10, 11, A3 (18), A4 (19) | 732 Hz
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 8]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+[options="header"]
+|================================================================================================================================================
+| Board                | PWM Pins                         | PWM Frequency                    
+| Uno, Nano, Mini      | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 5 and 6(980 Hz)
+| Mega                 | 2 - 13 and 44 - 46               | 490 Hz, exception pin 4 and 13(980 Hz)
+| Leonardo, Micro, Yún | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 3 and 11(980 Hz)                
+| Uno WiFi Rev.2       | 3, 5, 6, 9 and 10                | 490 Hz            
+| MKR boards           | 0, 1, 2, 3, 4, 5, 6, 7, 8, 10, A3 - or 18 -, A4 -or 19                           | 732 Hz                                
+| Zero                 | 3, 4, 5, 6, 8, 9, 10, 11, 12, 13 | 730 Hz
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| Zero *               | 3 - 13, A0 (14), A1 (15)        | 732 Hz
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 9]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+| Due                  | 3, 5, 6, 9, 10 and 11                | 1000 Hz
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| Due *                | 2-13                            | 1000 Hz
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 10]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+|================================================================================================================================================
+| Board                | PWM Pins                         | PWM Frequency                    
+| Uno, Nano, Mini      | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 5 and 6(980 Hz)
+| Mega                 | 2 - 13 and 44 - 46               | 490 Hz, exception pin 4 and 13(980 Hz)
+| Leonardo, Micro, Yún | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 3 and 11(980 Hz)                
+| Uno WiFi Rev.2       | 3, 5, 6, 9 and 10                | 490 Hz            
+| MKR boards           | 0, 1, 2, 3, 4, 5, 6, 7, 8, 10, A3 - or 18 -, A4 -or 19                           | 732 Hz                                
+| Zero                 | 3, 4, 5, 6, 8, 9, 10, 11, 12, 13 | 730 Hz
+| Due                  | 3, 5, 6, 9, 10 and 11                | 1000 Hz                   
+| 101                  | 3, 5, 6 and 9                      | 490 Hz
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
| 101                  | 3, 5, 6, 9                      | pins 3 and 9: 490 Hz, pins 5 and 6: 980 Hz
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 11]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+| Board                | PWM Pins                         | PWM Frequency                    
+| Uno, Nano, Mini      | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 5 and 6(980 Hz)
+| Mega                 | 2 - 13 and 44 - 46               | 490 Hz, exception pin 4 and 13(980 Hz)
+| Leonardo, Micro, Yún | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 3 and 11(980 Hz)                
+| Uno WiFi Rev.2       | 3, 5, 6, 9 and 10                | 490 Hz            
+| MKR boards           | 0, 1, 2, 3, 4, 5, 6, 7, 8, 10, A3 - or 18 -, A4 -or 19                           | 732 Hz                                
+| Zero                 | 3, 4, 5, 6, 8, 9, 10, 11, 12, 13 | 730 Hz
+| Due                  | 3, 5, 6, 9, 10 and 11                | 1000 Hz                   
+| 101                  | 3, 5, 6 and 9                      | 490 Hz                                
+|================================================================================================================================================
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
|====================================================================================================
* In addition to PWM capabilities on the pins noted above, the Due has true analog output when using `analogWrite()` on pins DAC0 and DAC1
* In addition to PWM capabilities on the pins noted above, the MKR boards and Zero have true analog output when using `analogWrite()` on the DAC0 (A0) pin.
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 12]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
 
 [float]
 === Description
-Writes an analog value (http://arduino.cc/en/Tutorial/PWM[PWM wave]) to a pin. Can be used to light a LED at varying brightnesses or drive a motor at various speeds. After a call to `analogWrite()`, the pin will generate a steady square wave of the specified duty cycle until the next call to `analogWrite()` (or a call to `digitalRead()` or `digitalWrite()`) on the same pin. The frequency of the PWM signal on most pins is approximately 490 Hz. On the Uno and similar boards, pins 5 and 6 have a frequency of approximately 980 Hz.
+Writes an analog value (http://arduino.cc/en/Tutorial/PWM[PWM wave]) to a pin. Can be used to light a LED at varying brightnesses or drive a motor at various speeds. After a call to `analogWrite()`, the pin will generate a steady square wave of the specified duty cycle until the next call to `analogWrite()` (or a call to `digitalRead()` or `digitalWrite()`) on the same pin. 
+[options="header"]
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion

[options="header"]
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 13]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+| Uno, Nano, Mini      | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 5 and 6(980 Hz)
+| Mega                 | 2 - 13 and 44 - 46               | 490 Hz, exception pin 4 and 13(980 Hz)
+| Leonardo, Micro, Yún | 3, 5, 6, 9, 10 and 11                | 490 Hz, exception pin 3 and 11(980 Hz)                
+| Uno WiFi Rev.2       | 3, 5, 6, 9 and 10                | 490 Hz            
+| MKR boards           | 0, 1, 2, 3, 4, 5, 6, 7, 8, 10, A3 - or 18 -, A4 -or 19                           | 732 Hz                                
+| Zero                 | 3, 4, 5, 6, 8, 9, 10, 11, 12, 13 | 730 Hz
+| Due                  | 3, 5, 6, 9, 10 and 11                | 1000 Hz                   
+| 101                  | 3, 5, 6 and 9                      | 490 Hz                                
+|================================================================================================================================================
+
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
```

File: Language/Functions/Analog IO/analogWrite.adoc [hunk 14]
```diff
@@ -17,10 +17,23 @@ subCategories: [ "Analog I/O" ]
...
+
```
Comment (per1234) [inline Language/Functions/Analog IO/analogWrite.adoc]:
```suggestion
```

**Comment by animeshsrivastava24** [comment]
@per1234 Pardon me for inefficiency in Pull Requests, it's lately been since I tried using the Git Version directly from Websites rather than using the command line, I'll divert back to the command line tool. I tried resolving above suggestions.

**Comment by per1234** [comment]
@animeshsrivastava24 I prefer to use a Git client software on my computer combined with Git commands from the command line for the things that the Git client doesn't do, or doesn't do well. However, the more limited GitHub web interface should work well enough for contributions to this project if people prefer that. Certainly the GitHub web interface is the easiest to use for people without prior experience with Git.

**Comment by animeshsrivastava24** [comment]
@per1234  Thank you so much for the guidance. Also, I am eagerly waiting for your e-mail replies so that I can summarize and progress more for the Documentation and GSoD'19 Work, please guide.

**Pull request merged**